### PR TITLE
Fix broken link for "Install the Weave Net addon" documentation in "Weave Net for NetworkPolicy" page

### LINKED ISSUE #45257: Broken link to "Install the Weave Net addon" documentation in "Weave Net for NetworkPolicy" page
The following links for Weave Net are no longer viable, and should either be updated (e.g. pointing to the GitHub repo where it is now maintained, https://github.com/weaveworks/weave, which again, admittedly needs to be updated, but puts the onus on the project itself), or the page should be completely removed:

> Install the Weave Net addon
> Follow the [Integrating Kubernetes via the Addon](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) guide.
> 
> The Weave Net addon for Kubernetes comes with a [Network Policy Controller](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/#npc) that automatically monitors Kubernetes for any NetworkPolicy annotations on all namespaces and configures iptables rules to allow or block traffic as directed by the policies.

Google link-to-highlight: https://kubernetes.io/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy/#:~:text=Install%20the%20Weave,by%20the%20policies.

This issue resembles #45201

### PR DESCRIPTION
**Pull request by cShirley14**
Updated links related to Weave Net documentation to placing onus of upkeep on Weave Net's github repository.
Closes #45257

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ You need to have a Kubernetes cluster. Follow the
 
 ## Install the Weave Net addon
 
-Follow the [Integrating Kubernetes via the Addon](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) guide.
+Follow the [Integrating Kubernetes via the Addon](https://github.com/weaveworks/weave/blob/master/site/kubernetes/kube-addon.md#-installation) guide.
 
 The Weave Net addon for Kubernetes comes with a
-[Network Policy Controller](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/#npc)
+[Network Policy Controller](https://github.com/weaveworks/weave/blob/master/docs/weavenpc-design.md)
```
[the remark this answers is not in the document]
Comment (cShirley14) [reply content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md]: That makes total sense, thanks for the assist!

**Review by bboreham — APPROVED** [review]
lgtm

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Comment by kingdonb** [comment]
I missed this discussion and commented here instead:

https://github.com/kubernetes/website/pull/45202#issuecomment-1976819974

There is a maintained fork of Weave Net now, it's rajch/weave and the new one-liner can be found here:

https://github.com/rajch/weave#using-weave-on-kubernetes

I will understand if there's apprehension about re-pointing the link at a personal fork, but there is no other active maintainer who has volunteered and done the work AFAIK. (Thanks @rajch!)

**Pull request merged**